lib/shopify_cli/messages/messages.rb

```diff
@@ -17,6 +17,7 @@ module Messages
         errors: {
           option_parser: {
             invalid_option: "The option {{command:%s}} is not supported.",
+            invalid_option_store_equals: "The option {{command:%s}} is not supported, did you mean: {{command:--store=%s}}.",
```

Comment:
The Shopify CLI UI repo actually has a component for similar scenarios. This one: 
![image](https://user-images.githubusercontent.com/51347120/150303140-a2c68796-e63f-4e08-aace-984ff0311f4d.png)

My recommendation:
- Use the above component (which includes "Did you mean?" in the frame)
- Content for inside the frame: {{command:--store=%s}}

Reply:
Thanks @MeredithCastile for the suggestion.

Unfortunately it seems this frame is specifically used in `cli-kit` under [very specific circumstances](https://github.com/Shopify/cli-kit/blob/32dc5707e1af5d2f99c589fc273dd9a9ca7ab5a3/lib/cli/kit/resolver.rb#L52), and isn't made very available to anything besides a total failure to resolve the command. Even option failures don't support the "Did you mean?" box.

We also can't change the format of output when raising `CLI::Abort` errors, as whatever we raise will be framed in the usual error box. This restriction is also imposed by `cli-kit`.

Since our output is generally bound by `cli-kit`, we don't have a lot of room to wiggle here without making changes to the underlying skeleton of the CLI.

We could change how errors work in `cli-kit`, and give the option to pass a did-you-mean suggestion, but this seems to be getting too into the weeds for a quick win.

So I'd recommend working under the assumption that the "Did you mean?" box isn't an option, and we should just do our best with the current error box.

Reply:
@amcaplan 

I'm definitely thinking about the CLI UI Kit. In the meantime, what about this as the error?

<img width="470" alt="image" src="https://user-images.githubusercontent.com/51347120/154320754-ce073443-7458-4fff-a35c-86ce3ff57a69.png">